creepy/utils/tests/test_processify.py

```diff
@@ -0,0 +1,47 @@
+import os
+import time
+import signal
+import multiprocessing
+from threading import Thread
+
+import pytest
+
+from ..processify import processify
+
+
+def test_processify_big_o_1():
```

Comment:
Shouldn't it measure algorithm complexity? Or it does and I don't understand that?

Reply:
Fixed name.